### ISSUE TITLE
Parsing of error response fails 

### ISSUE THREAD

**Issue by mwesox** [rockset/rockset-java-client#30]
You try to parse your own error response as JSON and it fails. 

```
com.rockset.client.ApiException: Bad Request
	at com.rockset.client.ApiClient.handleResponse(ApiClient.java:905)
	at com.rockset.client.ApiClient$1.onResponse(ApiClient.java:857)
	at com.squareup.okhttp.Call$AsyncCall.execute(Call.java:177)
	at com.squareup.okhttp.internal.NamedRunnable.run(NamedRunnable.java:33)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "message_key" (class com.rockset.client.model.ErrorModel), not marked as ignorable (7 known properties: "error_id", "line", "trace_id", "column", "message", "type", "query_id"])
 at [Source: (String)"{"message":"Document is not a valid JSON Object.","message_key":"INVALID_JSON_DOC","type":"InvalidInput","line":null,"column":null,"trace_id":"4a84e4ec-8f92-4021-aa0e-c1f7b4bf6457","error_id":null,"query_id":null}"; line: 1, column: 66] (through reference chain: com.rockset.client.model.ErrorModel["message_key"])
	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1127)
```

![image](https://user-images.githubusercontent.com/27929023/193409176-7ae80821-30ee-4e87-bee3-52d1183af567.png)

**Comment by giacomo-calixa** [comment]
I also confirm this:

```
rockset.com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "message_key" (class com.rockset.client.model.ErrorModel), not marked as ignorable (7 known properties: "error_id", "line", "trace_id", "column", "message", "type", "query_id"])
 at [Source: (String)"{"message":"Internal Error: The query failed to execute successfully. Please try again. If this query continues to fail, please contact customer support via Live Chat for further assistance.","message_key":null,"type":"InternalError","line":null,"column":null,"trace_id":"c25149d4-3f6d-4ff9-bd9b-e7d1757381b3","error_id":"","query_id":"c25149d4-3f6d-4ff9-bd9b-e7d1757381b3:Uh3gfQF:0"}"; line: 1, column: 211] (through reference chain: com.rockset.client.model.ErrorModel["message_key"])
	at rockset.com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
	at rockset.com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1127)
	at rockset.com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:1989)
	at rockset.com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1700)
	at rockset.com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1678)
	at rockset.com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:319)
	at rockset.com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:176)
	at rockset.com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:322)
	at rockset.com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4674)
	at rockset.com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3629)
	at rockset.com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3597)
	at com.rockset.client.ApiClient.handleResponse(ApiClient.java:901)
```
client version `rockset-java:0.9.2`

**Comment by kwadhwa18** [comment]
fix has been released in release [0.9.4](https://github.com/rockset/rockset-java-client/releases/tag/0.9.4). Thank you for reporting!
